dali/python/nvidia/dali/experimental/eager/__init__.py

```diff
@@ -67,4 +68,23 @@ def __enter__(self):
     def __exit__(self, type, value, traceback):
         arithmetic._enabled = self.prev
 
+    __name__ = 'arithmetic'
     _enabled = False
+
+
+class rng_state(_create_module_class()):
+    """ Manager class for stateful operators. Methods of this class correspond to the appropriate
+    functions in the fn API, they are created by :func:`_wrap_stateful` and are added dynamically.
```

Comment:
`they are created by :func:`_wrap_stateful` and are added dynamically.` - this part should probably be a comment, and the docstring for the class or for the `__init__` could add some info about the seeds, and some rough sketch of what this state promises.

Reply:
done